wolfcrypt/src/sp_arm64.c

```diff
@@ -43595,7 +43595,7 @@ static void sp_384_proj_point_dbl_n_6(sp_point_384* p, int n,
     x = p->x;
     y = p->y;
     z = p->z;
-
+    volatile int n = i;
```

Comment:
Variable declarations must be higher up. Can you check if having volatile in the second argument also works?